### ISSUE TITLE
Staging Review finding: Unable to determine focus between form steps

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#114655]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Rep Facing
**Product:** Accredited Rep Portal
**Feature:** Claims Form Upload

## Findings details
**VA.gov Experience Standard - issue:** [User can identify an element and its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When navigating between form steps, focus is either manually set or focus is stuck on the previous page's button. It doesn't look to be set to an element with a text label and there is no visible focus outline.

When navigating via keyboard, the user won't know where they are on the page and where their next Tab will go.
When using a screen reader, the user will not know the page has changed or where they are on it.

<details><summary>Link, screenshot or steps to recreate</summary>

All steps of form flow.

Video shows Safari with VoiceOver.

https://github.com/user-attachments/assets/2f15e40a-21c4-417f-a190-096fb050392e

</details>

## Recommended action

Manually setting focus is okay, but it should be set to a meaningful element. In this case, the headings under the segmented progress bar would be the most meaningful. These will be read out by a screen reader when progressing between steps.

Also ensure the focus outline styles are applied.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by jquispe-oddball** [comment]
@candilemoine @lauraparadis-nava you should be able to see a focus on the heading section of each step

**Comment by jquispe-oddball** [comment]
When the user goes to the next page, the focus will go to the h1, similar to how we handle the poa requests details page with the form.
cc : @candilemoine @lauraparadis-nava
